hil-test/tests/esp_wifi_init.rs

```diff
@@ -0,0 +1,65 @@
+//! Test we get an error when attempting to initialize esp-wifi with interrupts
+//! disabled in common ways
```

Comment:
Does this include initializing esp-wifi in an interrupt executor?

Reply:
I think I never even tried to use esp-wifi from an interrupt executor controlled task - did that ever work? If not we should document that

Reply:
Right now you can't initialize esp_wifi, I think, but with the builtin scheduler it's probably for the best anyway.